addons/overheating/CfgVehicles.hpp

```diff
@@ -66,7 +66,7 @@ class CfgVehicles {
                 };
                 class GVAR(CoolWeaponWithItem) {
                     displayName = CSTRING(CoolWeaponWithItem);
-                    condition = QUOTE(GVAR(enabled) && {isClass (configfile >> 'CfgPatches' >> 'acex_field_rations')});
+                    condition = 'GVAR(enabled) && {[_player, _target] call FUNC(canCoolWeaponWithItem)} && {!(_target getVariable [QEGVAR(captives,isSurrendering), false])} && {!(_target getVariable [QEGVAR(captives,isHandcuffed), false])}';
```

Comment:
It seems like this actually compiles with pboProject,
but with dev (build.py/hemtt) builds it crashes game with
```
 File z\ace\addons\overheating\CfgVehicles.hpp, line 68: '/CfgVehicles/CAManBase/ACE_Actions/ACE_Weapon/ace_overheating_CoolWeaponWithItem.condition': Missing ';' prior '}'
 Error context & {!(_target getVariable ["ace_captives_isSurrendering", false])} && {!(_target getVariable ["ace_captives_isHandcuffed", false])}';
 ErrorMessage: File z\ace\addons\overheating\CfgVehicles.hpp, line 68: '/CfgVehicles/CAManBase/ACE_Actions/ACE_Weapon.': '&' encountered instead of '='
```